tests/network/jumbo_frame/test_bond.py

```diff
@@ -141,7 +142,9 @@ def bond_bridge_attached_vma(
     name = "bond-vma"
     networks = OrderedDict()
     networks[br1bond_nad.name] = br1bond_nad.name
-    network_data_data = {"ethernets": {"eth1": {"addresses": ["10.200.1.1/24"]}}}
+    network_data_data = {
+        "ethernets": {"eth1": {"addresses": [f"{random_ipv4_address(net_seed=0, host_address=1)}/24"]}}
```

Comment:
Is it safe to use seed 0? I see it being used in the `util` already. I am unsure about the relation, but keeping it in sync with the 3rd octet could have been safer.

As this has been verified, I tend to consider this less critical, otherwise you should have seen in failing.

Reply:
Ok. But starting from 0 is IMO risky as mentioned before, so try to avoid it in the future.
You can resolve this thread.